CONTRIBUTING.md

```diff
@@ -30,34 +30,73 @@ Follow the steps below when submitting your code.
 
 1. Search the [list of issues](https://github.com/DevExpress/testcafe-hammerhead/issues) to see if there is an issue for the bug or feature you are going to work on or create a new one.
 
-2. If you are going to address an existing issue, check the comment thread to make sure that nobody is working on it at the moment.
+2. To address an already described issue, check the comment thread to make sure that nobody is working on it at the moment. Leave a comment saying that you are willing to fix this issue, and include details on how you plan to do this. Core team members may need to discuss the details of the proposed fix with you. After the green light from them,
+leave a comment saying that you started your work on this issue.
 
-3. Leave a comment saying that you are willing to fix this issue, and if possible, provide details on how you are going to do this.
+3. Install [Node.js](https://nodejs.org/en/) on your development machine.
 
-4. Core team members may need to discuss the details of the proposed fix with you. As soon as you get the green light from them,
-  leave a comment saying that you are currently working on this issue.
-
-5. Fork TestCafe Hammerhead and create a branch in your fork. Name this branch with an issue number, for example `gh852`, `gh853`.
+4. Fork TestCafe Hammerhead and create a branch in your fork. Name this branch with an issue number, for example, `gh852`, `gh853`.
   
-6. Commit your changes into the branch.
+5. Install dependencies. In the root directory of your local copy, run:
+
+    ```sh
+    npm install
+    ```
+
+    or (for [Yarn](https://yarnpkg.com/) users):
+
+    ```sh
+    yarn
+    ```
+
+6. Write code and commit your changes to the branch.
 
-7. Add regression tests to the appropriate sections if you are fixing a bug. You can find these sections by searching for `Regression` in the code.
+    To build and launch TestCafe Hammerhead, run the `http-playground` task:
+
+    ```sh
+    node node_modules/gulp/bin/gulp http-playground
+    ```
 
-    Add unit and/or functional tests if you are developing a new functionality.
+    The `http-playground` task builds Hammerhead, launches a local HTTP server, and opens a playground page, where you can specify a web page to proxy.
+
+    To run a playground over HTTPS, run the `https-playground` task.
+
+    ```sh
+    node node_modules/gulp/bin/gulp https-playground
```

Comment:
Do we need to launch gulp directly from `bin`? Can't we do this instead:

```sh
 node gulp https-playground
 ```

Reply:
You can make it even simpler:

`gulp https-playground`

See step 9 -- there's a short command used.